### PR TITLE
Kazakhstan: update holidays relevance period and names

### PR DESCRIPTION
**Pull request by KJhellico**

### CODE REVIEW

**Comment by mihalken** [comment]
Hello. I'm already working on it. :) https://github.com/dr-prodigy/python-holidays/pull/864
I have suggestions...

File: holidays/countries/kazakhstan.py [hunk 1]
```diff
@@ -13,81 +13,92 @@
...
-
-        # Unity Day
-        self[date(year, MAY, 1)] = "Unity Day"
-
-        # Defender's Day
+        name = "New Year"
+        self[date(year, JAN, 1)] = name
+        self[date(year, JAN, 2)] = name
+
+        # International Women’s Day
```
Comment (arkid15r) [inline holidays/countries/kazakhstan.py:39]: Not sure whether ' (vs ’) is more standard option.

**Review by arkid15r — COMMENTED** [review]
LGTM

File: holidays/countries/kazakhstan.py (hunk 1 above)
Comment (KJhellico) [reply holidays/countries/kazakhstan.py]: Thank you. Replaced with '

**Pull request merged**